.circleci/config.yml.in

```diff
@@ -152,11 +152,10 @@ commands:
           args: --no-build-isolation <<# parameters.editable >> --editable <</ parameters.editable >> .
           descr: Install torchvision <<# parameters.editable >> in editable mode <</ parameters.editable >>
 
+  # Installs all extra dependencies that are needed in the torchvision.prototype namespace, but are not tracked in the
+  # project requirements.
   install_prototype_dependencies:
     steps:
-      - pip_install:
-          args: iopath
-          descr: Install third-party dependencies
```

Comment:
For some time now `iopath` is no longer a dependency at import and is only used at runtime in

https://github.com/pytorch/vision/blob/93104c1697384273a75381d57be2f872265eeefb/torchvision/prototype/datasets/utils/_internal.py#L175-L183

To my best knowledge, we are currently not using that. Thus, it should be safe to remove here.

---

I was only looking at this, because I was adding the `dill` installation. LMK, if I should bring a separate PR for this change.